<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewAdvancedSettingsDialog</class>
 <widget class="QDialog" name="PreviewAdvancedSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview Advanced Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0">
    <widget class="QLabel" name="label_text">
     <property name="text">
      <string>Settings below may be overriden by frame properties.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Bicubic filter parameter b:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="yuvMatrixCoefficientsComboBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="chromaPlacementComboBox"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="chromaResamplingFilterComboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Chroma resampling filter:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Y'CbCr / Y'UV matrix coefficients:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Chroma placement:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Lanczos filter taps:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Bicubic filter parameter c:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="bicubicFilterParameterCSpinBox">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000001000000000</double>
     </property>
     <property name="value">
      <double>0.333333000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="bicubicFilterParameterBSpinBox">
     <property name="decimals">
      <number>6</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.000001000000000</double>
     </property>
     <property name="value">
      <double>0.333333000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetToDefaultButton">
       <property name="text">
        <string>Reset to default</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_dither">
     <property name="text">
      <string>Dither type:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="ditherTypeComboBox"/>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="lanczosFilterTapsSpinBox">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>4096</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="silentSnapshotCheckBox">
     <property name="text">
      <string>Silently save snapshots with template:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QToolButton" name="argumentsHelpButton">
     <property name="text">
     </property>
     <property name="icon">
      <iconset resource="../../../resources/vsedit.qrc">
      <normaloff>:/information.png</normaloff>:/information.png</iconset>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLineEdit" name="saveSnapshotTemplateLineEdit">
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
